bodhi/models/models.py

```diff
@@ -706,16 +707,6 @@ def new(cls, request, data):
                     build.package for build in data['builds']
                 ] if pkg.requirements], []))))
 
-        if not data['autokarma']:
-            del(data['stable_karma'])
-            del(data['unstable_karma'])
-            caveats.append({
-                'name': 'autokarma',
-                'description': 'Auto-karma stable requests disabled.',
-            })
-
-        del(data['autokarma'])
```

Comment:
Yet, in the deployed Bodhi 2.2.2, it still does not remember the karma values if autokarma is disabled. There must be some more code doing that.

This and the other issue I pointed out in this review make the feature from pull request #821 entirely non-functional.